### DISCUSSION TITLE
⏳ 🔗 Can Elementor have arrows in the "Taxonomy Filter" to scroll better? This is because of the active "Horizontal Scroll" option

### DISCUSSION THREAD

**Discussion by PachacutecInca** [elementor/elementor#24692] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

![image](https://github.com/elementor/elementor/assets/49765800/1bcf8df1-bf39-4f09-a172-adbe54d4f05e)

The scroll is not very intuitive

### Describe the solution you'd like.

Arrows in both sides like slides

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by StachRedeker** [comment]
+1. This would be a great quality of life update.

**Comment by chrisp2u** [comment]
Yes, please. Awful UI without some kind of visual indicator.
